src/css/base.scss

```diff
@@ -62,17 +145,106 @@ inset -2px -2px 3px rgba(0, 0, 0, .6);
               inset 2px 2px 3px rgba(0, 0, 0, .6);
 }
 
-#puzzle-table {
-  margin-left: 33%;
-  height: 60%;
+.puzzle-table__display {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+}
+
+.board-column__section 
+{
+  display: flex;
+  flex-direction: column;
+  /* justify-content: space-around; */
+  width: 20%;
+}
+
+.column-row__display
+{
+  margin: 0% 0% 1% 1%;
+  height: 25%;
+  border-radius: 7%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: end;
+  font-size: 42px;
+  font-weight: 800;
+  color: goldenrod;
+  border-style: solid;
+  border-width: 5px;
+  border-color: black;
+  background-color: navy;
+  opacity: 0.95;
+  filter: drop-shadow(2px 4px 6px black);
+}
+
+.column-header__category {
+  margin: 0% 0% 1% 1%;
+  height: 12%;
+  border-radius: 7%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: end;
+  font-size: 23px;
+  font-weight: 800;
+  color: white;
+  border-style: solid;
+  border-width: 5px;
+  border-color: black;
+  background-color: navy;
+  opacity: 0.95;
+  filter: drop-shadow(2px 4px 6px black);
+}
+
+.player-scorecard {
+  height: 12%;
+  width: 16%;
+  border-radius: 7%;
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  // align-content: end;
+  font-size: 23px;
+  font-weight: 800;
+  color: white;
+  border: black solid 5px;
+  // border-style: solid;
+  // border-width: 5px;
+  // border-color: black;
+  background-color: navy;
+  opacity: 0.95;
+  filter: drop-shadow(2px 4px 6px black);
+}
+
+.alert-question__display {
+  width: 90%;
+  height: 850px;
+  border-radius: 7%;
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  height: 750px;
+  border-radius: 7%;
+  flex-direction: column;
+  display: flex;
+  /* justify-content: center; */
+  align-content: center;
+  font-size: 42px;
+  font-weight: 800;
+  color: goldenrod;
+  border-style: solid;
+  border-width: 5px;
+  border-color: black;
+  background-color: navy;
+  opacity: 0.95;
+  filter: drop-shadow(2px 4px 6px black);
```

Comment:
Any chance this option could be contributing to the jumpy image load we witnessed? Not that I'm suspect of this because of previous experience, just reaching a little to try to figure out why it's acting the way it is.